deployment/CP03ISSM_Deploy.csv

```diff
@@ -1,343 +1,374 @@
+AR48,,,,CP03ISSM-MFD35-05-PCO2WB000,12,1,2020-10-28T00:00:00,,CGMCP-03ISSM-00012,,CGINS-PCO2WB-C0100,40,-70,,,,
+AR48,,,,CP03ISSM-MFD35-06-PHSEND000,12,1,2020-10-28T00:00:00,,CGMCP-03ISSM-00012,,CGINS-PHSEND-P0158,40,-70,,,,
+AR48,,,,CP03ISSM-MFD37-00-DCLENG000,12,1,2020-10-28T00:00:00,,CGMCP-03ISSM-00012,,CGCON-MDCL37-00007,40,-70,,,,
+AR48,,,,CP03ISSM-MFD37-01-OPTAAD000,12,1,2020-10-28T00:00:00,,CGMCP-03ISSM-00012,,CGINS-OPTAAD-00130,40,-70,,,,
+AR48,,,,CP03ISSM-MFD37-03-CTDBPD000,12,1,2020-10-28T00:00:00,,CGMCP-03ISSM-00012,,CGINS-CTDBPD-50110,40,-70,,,,
+AR48,,,,CP03ISSM-MFD37-04-DOSTAD000,12,1,2020-10-28T00:00:00,,CGMCP-03ISSM-00012,,CGINS-DOSTAD-00388,40,-70,,,,
+AR48,,,,CP03ISSM-MFD37-07-ZPLSCC000,12,1,2020-10-28T00:00:00,,CGMCP-03ISSM-00012,,CGINS-ZPLSCC-55101,40,-70,,,,
+AR48,,,,CP03ISSM-RIC21-00-CPMENG000,12,1,2020-10-28T00:00:00,,CGMCP-03ISSM-00012,,CGCON-NCPM02-00033,40,-70,,7,,
+AR48,,,,CP03ISSM-RID26-00-DCLENG000,12,1,2020-10-28T00:00:00,,CGMCP-03ISSM-00012,,CGCON-NDCL26-00006,40,-70,,7,,
+AR48,,,,CP03ISSM-RID26-04-VELPTA000,12,1,2020-10-28T00:00:00,,CGMCP-03ISSM-00012,,CGINS-VELPTA-11417,40,-70,,7,,
+AR48,,,,CP03ISSM-RID26-06-PHSEND000,12,1,2020-10-28T00:00:00,,CGMCP-03ISSM-00012,,CGINS-PHSEND-P0100,40,-70,,7,,
+AR48,,,,CP03ISSM-RID26-07-NUTNRB000,12,1,2020-10-28T00:00:00,,CGMCP-03ISSM-00012,,CGINS-NUTNRB-01065,40,-70,,7,,
+AR48,,,,CP03ISSM-RID26-08-SPKIRB000,12,1,2020-10-28T00:00:00,,CGMCP-03ISSM-00012,,CGINS-SPKIRB-00273,40,-70,,7,,
+AR48,,,,CP03ISSM-RID27-00-DCLENG000,12,1,2020-10-28T00:00:00,,CGMCP-03ISSM-00012,,CGCON-NDCL27-00033,40,-70,,7,,
+AR48,,,,CP03ISSM-RID27-01-OPTAAD000,12,1,2020-10-28T00:00:00,,CGMCP-03ISSM-00012,,CGINS-OPTAAD-00207,40,-70,,7,,
+AR48,,,,CP03ISSM-RID27-02-FLORTD000,12,1,2020-10-28T00:00:00,,CGMCP-03ISSM-00012,,CGINS-FLORTD-01216,40,-70,,7,,
+AR48,,,,CP03ISSM-RID27-03-CTDBPC000,12,1,2020-10-28T00:00:00,,CGMCP-03ISSM-00012,,CGINS-CTDBPC-50002,40,-70,,7,,
+AR48,,,,CP03ISSM-RID27-04-DOSTAD000,12,1,2020-10-28T00:00:00,,CGMCP-03ISSM-00012,,CGINS-DOSTAD-00387,40,-70,,7,,
+AR48,,,,CP03ISSM-SBC11-00-CPMENG000,12,1,2020-10-28T00:00:00,,CGMCP-03ISSM-00012,,CGCON-BCPM01-50004,40,-70,,0,,
+AR48,,,,CP03ISSM-SBD11-00-DCLENG000,12,1,2020-10-28T00:00:00,,CGMCP-03ISSM-00012,,CGCON-BDCL11-50004,40,-70,,0,,
+AR48,,,,CP03ISSM-SBD11-01-MOPAK0000,12,1,2020-10-28T00:00:00,,CGMCP-03ISSM-00012,,CGINS-MOPAK0-11839,40,-70,,0,,
+AR48,,,,CP03ISSM-SBD11-02-HYDGN0000,12,1,2020-10-28T00:00:00,,CGMCP-03ISSM-00012,,CGINS-HYDGN0-41692,40,-70,,0,,
+AR48,,,,CP03ISSM-SBD11-06-METBKA000,12,1,2020-10-28T00:00:00,,CGMCP-03ISSM-00012,,CGINS-METLGR-00033,40,-70,,0,,
+AR48,,,,CP03ISSM-SBD12-00-DCLENG000,12,1,2020-10-28T00:00:00,,CGMCP-03ISSM-00012,,CGCON-BDCL12-50004,40,-70,,0,,
+AR48,,,,CP03ISSM-SBD12-03-HYDGN0000,12,1,2020-10-28T00:00:00,,CGMCP-03ISSM-00012,,CGINS-HYDGN0-49868,40,-70,,0,,
+AR48,,,,CP03ISSM-SBD12-04-PCO2AA000,12,1,2020-10-28T00:00:00,,CGMCP-03ISSM-00012,,CGINS-PCO2AA-33182,40,-70,,0,,
```

Comment:
Looks good. Reviewed CUIDs and serial numbers with config sheet

Reply:
Looks good.